packages/v4/src/content/design-guidelines/components/switch/switch.md

```diff
@@ -44,3 +44,40 @@ A switch changes an option and saves it simultaneously, while checkboxes require
 * Use checkboxes when you may have an intermediate state where you can select all, none, or some actions.
 
     <img src="./img/switch-check-3.png" alt="Example 3 of do's and don'ts for using a checkbox vs. a switch" width="661"/>
+
+## Content 
+### Form labels
+When writing form labels, keep the message short and specific. Ensure that the message is clear when the switch is toggled to both the “on” and “off” position. 
+
+<div class="ws-content-table">
+
+| **Do** | **Don’t** |
+|:------:|:---------:|
+| Enabled encryption of installed disks| Encrypt installed disks|
```

Comment:
Based on the text below, I think you might have switched up the two examples (The Do should say "Encrypt installed disks")